services/twitter.js

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable no-console */
```

Comment:
thoughts on suffixing as a service to make it more apparent? `twitterService`. 
Smurf naming is always a fun debate. 

tbh, when I was reading `new Twitter` earlier, I wasn't sure what the class did. It would have been nice to know without looking it up

Reply:
I think that's a good name for the class. I dunno if we need to call the file that since it's already in `services/`.
I'll change the class name to TwitterService.

Reply:
brilliant!